src/test/java/org/jboss/windup/rules/tests/HintExists.java

```diff
@@ -24,7 +25,7 @@
 
     private HintExists(String messagePattern)
     {
-        this.messagePattern = messagePattern;
+        this.messagePattern = "[\\s\\S]*" + messagePattern + "[\\s\\S]*";
```

Comment:
<message> </message> introduced weird characters at the start/end of the messagePattern